openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditor.tsx

```diff
@@ -86,11 +81,8 @@ const RichTextEditor = forwardRef<editorRef, EditorProp>(
       getEditorContent(format: 'json' | 'markdown') {
         switch (format) {
           case Format.MARKDOWN:
-            return draftjsToMd(convertToRaw(editorState.getCurrentContent()));
-
-          case Format.JSON:
           default:
-            return convertToRaw(editorState.getCurrentContent());
+            return draftjsToMd(convertToRaw(editorState.getCurrentContent()));
```

Comment:
Remove switch case for now and add a comment for reference to use switch case if needed later

Reply:
@darth-coder00 , sure I will remove that.